client/src/pages/CalendarPage/CalendarPage.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import "react-calendar/dist/Calendar.css";
+import "./CalendarOverride.css";
+import Calendar from "react-calendar";
+import styles from "./CalendarPage.module.css";
+import TaskCard from "../../components/TaskCard/TaskCard.js";
+import CalendarHeader from "../../components/CalendarHeader/CalendarHeader.js";
+
+const dayLabelFormatter = (locale, label) => {
+  return label.toString().slice(0, 1);
+};
+
+const dailyTaskDateFormatter = (date) => {
+  const months = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+  ];
+  return String.raw`${
+    months[date.getMonth()]
+  } ${date.getDate()}, ${date.getFullYear()}`.toString();
```

Comment:
.toString() shouldn't be needed since String.raw should return a string

Reply:
Removed.